tests/unit/aiplatform/test_featurestores.py

```diff
@@ -1603,6 +1609,9 @@ def test_batch_serve_to_gcs_with_invalid_gcs_destination_type(self):
                 read_instances_uri=_TEST_GCS_CSV_SOURCE_URI,
             )
 
+    @pytest.mark.skipif(
+        _USE_BQ_STORAGE is False, reason="batch_serve_to_df requires bigquery_storage"
```

Comment:
(optional)

Another way to achieve this is `"bigquery_storage" not in  sys.modules", ...`

Up to your preference, since `sys` was not imported before.